### PR TITLE
Update images.json

### PR DESCRIPTION
**Pull request by Ajay84sia**
Added SVG Backgrounds

### CODE REVIEW

**Review by rupali-codes — COMMENTED** [review]
Could you please remove that extra line? line-50

**Comment by Ajay84sia** [comment]
done, extra line? line-50 removed

**Review by ujjawaltyagii — APPROVED** [review]
LGTM 👍

File: database/frontend/images.json [hunk 1]
```diff
@@ -39,6 +39,12 @@
         "url": "https://shotstash.com/",
         "category": "frontend",
         "subcategory": "images"
+    },
+     {
+        "name": "SVG Backgrounds",
+        "description": "This website provide you customize svg background for your projects",
```
Comment (CBID2) [inline database/frontend/images.json:45]:
```suggestion
        "description": "This website provides you customized SVG backgrounds for your projects",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Ajay84sia! 😄 I added a suggestion to make your description clearer! 😄 For future reference, I highly recommend using [Grammarly](https://www.grammarly.com/about). It's a great tool for proofreading and revising the content you write! 😄 Keep it up! 👍🏾

**Comment by Ajay84sia** [comment]
Thanks, @CBID2 for your suggestion. I will use Grammarly

**Review by CBID2 — APPROVED** [review]
Well done @Ajay84sia! 😄 LGTM! 👍🏾

**Comment by rupali-codes** [comment]
Thanks

**Pull request merged**